build_web_compilers/lib/src/dart2js_bootstrap.dart

```diff
@@ -24,15 +24,15 @@ import 'web_entrypoint_builder.dart';
 Future<void> bootstrapDart2Js(
   BuildStep buildStep,
   List<String> dart2JsArgs, {
-  required bool nativeNullAssertions,
+  required bool? nativeNullAssertions,
```

Comment:
[optional] Doesn't need to be required anymore

Reply:
I like making it required to be explicit even though it's nullable